interfaces/builtin/desktop.go

```diff
@@ -205,8 +205,8 @@ dbus (send)
 
 ## Allow access to xdg-document-portal file system.  Access control is
 ## handled by bind mounting a snap-specific sub-tree to this location.
-owner /run/user/[0-9]*/doc/ r,
-owner /run/user/[0-9]*/doc/** rw,
+owner /run/user/[0-9]*/doc/{,*} r,
```

Comment:
I was looking at this access and it seems that portals is creating `/run/user/.../doc/<tmpdir>` (`/run/user/.../doc/by-app/<snap name>/<tmpdir>` on the host), so shouldn't this access be:
```
owner /run/user/[0-9]*/doc/{,*/} r,
```

Reply:
That looks correct yes.  There should only be directories under $XDG_RUNTIME_DIR/doc, and that's what we want to give read access to.